### PR TITLE
[DOCU-1603] Add title attrs and more desc link titles

### PR DESCRIPTION
**Pull request by falondarville**
### Review

@Kong/team-docs 

### Summary

/hub:
* add `title` attributes to links with `extn.name` and `extn.desc`

/deck:
* change clickable video stills to easy-to-read links
* update links to be more readable and descriptive

/kubernetes-ingress-controller:
* update links to be more readable and descriptive

### Reason

Accessibility efforts. 

Note that for /decK, clickable images appear misleading (as hosted or imbedded media) for users not using a screen reader. For screen-reader users, there was no description of the image. I opted to add descriptive links instead. 

### Testing

Tested locally. 

https://deploy-preview-3017--kongdocs.netlify.app/hub/ 
https://deploy-preview-3017--kongdocs.netlify.app/deck/
https://deploy-preview-3017--kongdocs.netlify.app/kubernetes-ingress-controller/

### Next Steps

I recommend creating an accessible links section in our Contributing Guide so that we can update as we go. 
* links should be descriptive of the content linked to. Instead of "click here", write "specific name of the resource".
* if the link contains added content (like the hub panels, which has an img), add a descriptive `title` attribute to the `a` tag. 
* refrain from using video stills, as they can be misleading

Resource accessed to learn more about link accessibility: https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML

### CODE REVIEW

**Review by GayleNeumann — APPROVED** [review]
LGTM

File: app/deck/1.7.x/index.md [hunk 1]
```diff
@@ -65,26 +62,21 @@ The command line `--help` flag on the main command or a subcommand (like diff,
...
-[here](/deck/{{page.kong_version}}/reference/deck).
+[See a list of all commands available with decK](/deck/{{page.kong_version}}/reference/deck).
 
 ## Frequently Asked Questions (FAQs)
 
-You can find answers to FAQs [here](/deck/{{page.kong_version}}/faqs).
+[Access our FAQs page](/deck/{{page.kong_version}}/faqs).
 
-## Explainer video
+## Kong Summit motivation behind decK video
```
Comment (lena-larionova) [inline app/deck/1.7.x/index.md:71]:
```suggestion
## Video: Kong Summit motivation behind decK
```
Maybe like this? Reads slightly cleaner.

File: app/kubernetes-ingress-controller/1.3.x/index.md [hunk 2]
```diff
@@ -46,12 +46,12 @@ load configuration intended for other instances or other ingress controllers.
 
 ### Security
 
-Please refer to [this document](/kubernetes-ingress-controller/{{page.kong_version}}/concepts/security) to understand the
+Please refer to the [Security concepts guide](/kubernetes-ingress-controller/{{page.kong_version}}/concepts/security) to understand the
```
Comment (lena-larionova) [inline app/kubernetes-ingress-controller/1.3.x/index.md:49]:
```suggestion
Refer to the [Security concepts guide](/kubernetes-ingress-controller/{{page.kong_version}}/concepts/security) to understand the
```

File: app/kubernetes-ingress-controller/1.3.x/index.md [hunk 3]
```diff
@@ -22,20 +22,20 @@ A few custom resources are bundled with the {{site.kic_product_name}} to
...
+Please refer to the [deployment documentation](/kubernetes-ingress-controller/{{page.kong_version}}/concepts/deployment),
 which explains all the components
 involved and different ways of deploying them based on the use-case.
 
 ### High-availability and Scaling
 
 The {{site.kic_product_name}} is designed to scale with your traffic
 and infrastructure.
-Please refer to [this document](/kubernetes-ingress-controller/{{page.kong_version}}/concepts/ha-and-scaling) to understand
+Please refer to the [High-availability and Scaling guide](/kubernetes-ingress-controller/{{page.kong_version}}/concepts/ha-and-scaling) to understand
```
Comment (lena-larionova) [inline app/kubernetes-ingress-controller/1.3.x/index.md:38]:
```suggestion
Refer to the [High-availability and Scaling guide](/kubernetes-ingress-controller/{{page.kong_version}}/concepts/ha-and-scaling) to understand
```

File: app/kubernetes-ingress-controller/1.3.x/index.md [hunk 4]
```diff
@@ -22,20 +22,20 @@ A few custom resources are bundled with the {{site.kic_product_name}} to
...
 over the proxying behavior.
 
-Please refer to [custom resources][crd] concept document for more details.
+Please refer to [custom resources concept document][crd] for more details.
 
 ### Deployment Methods
 
 The {{site.kic_product_name}} can be deployed in a variety of deployment patterns.
-Please refer to the [deployment](/kubernetes-ingress-controller/{{page.kong_version}}/concepts/deployment) documentation,
+Please refer to the [deployment documentation](/kubernetes-ingress-controller/{{page.kong_version}}/concepts/deployment),
```
Comment (lena-larionova) [inline app/kubernetes-ingress-controller/1.3.x/index.md:30]:
```suggestion
Refer to the [deployment documentation](/kubernetes-ingress-controller/{{page.kong_version}}/concepts/deployment),
```

File: app/kubernetes-ingress-controller/1.3.x/index.md [hunk 5]
```diff
@@ -22,20 +22,20 @@ A few custom resources are bundled with the {{site.kic_product_name}} to
 configure settings that are specific to Kong and provide fine-grained control
 over the proxying behavior.
 
-Please refer to [custom resources][crd] concept document for more details.
+Please refer to [custom resources concept document][crd] for more details.
```
Comment (lena-larionova) [inline app/kubernetes-ingress-controller/1.3.x/index.md:25]:
```suggestion
Refer to the [custom resources concept document][crd] for more details.
```

File: app/kubernetes-ingress-controller/1.3.x/index.md [hunk 6]
```diff
@@ -46,12 +46,12 @@ load configuration intended for other instances or other ingress controllers.
...
 ### Security
 
-Please refer to [this document](/kubernetes-ingress-controller/{{page.kong_version}}/concepts/security) to understand the
+Please refer to the [Security concepts guide](/kubernetes-ingress-controller/{{page.kong_version}}/concepts/security) to understand the
 default security settings and how to further secure the Ingress Controller.
 
 ## Guides and Tutorials
 
-Please browse through [guides][guides] to get started or understand how to configure
+Browse through [guides][guides] to get started or understand how to configure
```
Comment (lena-larionova) [inline app/kubernetes-ingress-controller/1.3.x/index.md:54]:
```suggestion
Browse through the [how-to guides][guides] to get started or understand how to configure
```
Not sure if that's the best term, but I think this needs a qualifier. On its own, "the guides" sounds vague.

**Review by lena-larionova — APPROVED** [review]
LGTM with minor suggestions. Removed all the "please"s since you started with one.

Are you going to add link instructions in this PR, or follow up with them? I agree in that this info should be added to the style guide.

**Comment by falondarville** [comment]
> Are you going to add link instructions in this PR, or follow up with them? I agree in that this info should be added to the style guide.

I'll make this a separate task since I'd like to explore more accessibility content to add to our Contributing Guide.

**Pull request merged**